common/web/gesture-recognizer/README.md

```diff
@@ -0,0 +1,22 @@
+# Keyman:  Gesture Recognizer Module
+The Original Code is (C) 2022-2022 SIL International
+
+The Gesture Recognizer module is an internal component for all of Keyman's Web-based OSKs.  At this time, this is only referenced by
+KeymanWeb, seen within this repo at /web/.
+
+## Minimum Compilation Requirements
+
+* A local installation of [Node.js](https://nodejs.org/) v8.9+.
```

Comment:
We should say node 16+ as we have that as our baseline for KMW. That avoids unnecessary back-compat reqs and testing. (Although I wonder if we omit the version here, as we have it in the setup docs, and it just becomes another place to update in the future)

Reply:
For this and the other currently-not-marked-as-resolved comments, I opted to instead insert a reference to the setup doc itself.  Is what I did there fine?  (Feel free to mark as "resolved" if so.)